docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md

```diff
@@ -136,7 +136,7 @@ nebula> BALANCE ID 1570761786
 
 ### Step 4 查看结果
 
-大多数情况下，搬迁数据是个比较漫长的过程。但是搬迁过程不会影响已有服务。运行结束后，进度会提示100%。如果有运行失败的 task，可再次运行 `balance data` 命令进行修复。如果多次运行仍无法修复，请与社区联系 [GitHub](https://github.com/vesoft-inc/nebula/issues)。最后，通过 `SHOW HOSTS` 查看运行后的 partition 分布。
+大多数情况下，搬迁数据是个比较漫长的过程。但是搬迁过程不会影响已有服务。运行结束后，进度会提示 100%。如果有运行失败的 task，可再次运行 `balance data` 命令进行修复。如果多次运行仍无法修复，请与社区联系 [GitHub](https://github.com/vesoft-inc/nebula/issues)。最后，通过 `SHOW HOSTS` 查看运行后的 partition 分布。
```

Comment:
BALANCE DATA

Reply:
changed all command to uppercase fix width